go.php

```diff
@@ -4,27 +4,27 @@
 ############## Adjust when editing docs ##################
 
 $mapping = array(
-    'admin-background-jobs'   => '/admin_manual/configuration_server/background_jobs_configuration.html',
+    'admin-background-jobs'   => '/admin_manual/configuration/server/background_jobs_configuration.html',
     'admin-dir_permissions'   => '/admin_manual/installation/installation_wizard.html#setting-strong-directory-permissions',
```

Comment:
New Label for the anchor: /admin_manual/installation/installation_wizard.html#post-installation-steps-label